tgeraser/core.py

```diff
@@ -53,18 +55,26 @@ async def main() -> None:
     arguments = docopt(__doc__, version=VERSION)
     if arguments["--limit"]:
         arguments["--limit"] = cast_to_int(arguments["--limit"], "limit")
+
+    periods = {
+        "seconds": 1,
+        "minutes": 60,
+        "hours": 3600,
+        "days": 86400,
+        "weeks": 604800,
+    }
     if arguments["--time-period"]:
-        periods = {
-            "seconds": 1,
-            "minutes": 60,
-            "hours": 3600,
-            "days": 86400,
-            "weeks": 604800,
-        }
         period = arguments["--time-period"].split("*")
         if period[1] not in periods:
             raise TgEraserException("Time period is specified incorrectly.")
```

Comment:
Instead of `Time period is specified incorrectly.`, it would be better to use a formulation like `'--time-period' argument specified incorrectly.`. A similar formulation should be used for the new option as well.